### ISSUE TITLE
[a11y] Cannot set user password

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1506]
![msedge_4PzfjFYZ3F](https://user-images.githubusercontent.com/37256050/167705626-32b3e13d-c06c-4a60-b88a-7e73c46ef341.gif)

![image](https://user-images.githubusercontent.com/37256050/167705642-d923bde4-456a-4b5a-aa8e-0ecbf6dc8d1b.png)

Password has since been assigned in SQL

db: `oldfish`

**Comment by maxxxxxdlp** [comment]
The password is only set after you press the "Save" button at the bottom of the screen.

There was a related bug where setting password did not enable the button, but that is fixed in b066f90d
